app/src/shared/components/error-boundary/error-boundary.stories.tsx

```
@@ -9,12 +9,12 @@ const meta: Meta = {
       children: 'throw new Error("Custom error example")',
     },
   }),
-  render: function Story(props: ReactProps) {
+
+  render: (props: ReactProps) => {
     const ThrowError = () => {
       eval(props.children as string)
```

Biome (2.4.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)